tasks/update.yml

```diff
@@ -1,13 +1,22 @@
 ---
-- name: Run database updates.
-  command: "{{ drush_path }} updatedb -y"
+
+- name: Detect Drupal version.
+  command: drush status drupal-version --format=list
   args:
     chdir: "{{ drupal_core_path }}"
+  register: drupal_core_version_output
+
+- name: Set Drupal version.
+  set_fact:
+    drupal_core_version: "{{ drupal_core_version_output.stdout }}"
 
-# TODO: Import configuration if configured?
-# TODO: Other commands if configured?
+- name: Run post-update tasks.
+  command: "{{ item.command }}"
+  args:
+    chdir: "{{ item.chdir | default(omit) }}"
```

Comment:
Personally I think `chdir` could be hardcoded to `drupal_core_path` without an option to change. I doubt anyone would ever needs to run anything from the home directory (not even sure if it's root or vagrant).

Reply:
@oxyc ^^

Reply:
@oxyc @geerlingguy 

Any chance of getting this merged after the conflicts are resolved?

Regarding `chdir`: The current code uses `chdir` anyway, so the PR is consistent with the current code in that regard.